pkg/promlib/go.mod

```diff
@@ -0,0 +1,110 @@
+module github.com/grafana/grafana/pkg/promlib
+
+go 1.21.0
+
+require (
+	github.com/grafana/grafana-azure-sdk-go v1.12.0
```

Comment:
I think you can easily remove this dependency (so the shared lib doesn't depend on the Azure SDK). It seems it's only using a helper function called "maputil", which I guess can be copied here or replicated.

Reply:
SGTM

Reply:
https://github.com/grafana/grafana-plugin-sdk-go/pull/919

Reply:
you can release a new SDK version and update this, no?

Reply:
Yes, I can do it. ;)

Reply:
https://github.com/grafana/grafana/pull/83952/commits/f31ef914483ac205b8c45a27cbb4dbac9c8875aa